### DISCUSSION TITLE
Ambient Sounds

### DISCUSSION THREAD

**Discussion by odvigajlo** [niels747/2D-Weather-Sandbox#60] [Ideas]
well title says... return ambient sounds!
Ambient sounds made game really peaceful a day when they're were added!
But then niels removed them because they were useless, but they should return back!

Pretty much conditions for different ambient sounds:
Wind 2 km/h - 20 km/h - Wind sounds (can be combined with other sounds)
Wind 20 km/h+ - Powerful wind sounds (can be combined with other sounds)
Little/Brittle Precipitation - Rain/Snow sounds (depending on temperature, can be combined with other sounds)
Medium Precipitation - Gusty Rain sounds
High/Extreme Precipitation - Extreme Rain sounds + Light Hail sounds
Calm + Zoomed near land - Bird sounds
Calm + Zoomed near ocean - Wave sounds

**Comment by niels747** [comment]
I like the idea, but it's difficult to implement a realistic sound system that reacts to the local weather conditions. I will add it to TODO

**Comment by coolkid0281** [comment]
20 km/h is like a little slow lol I think like 50 km/h for powerful wind sounds because 20 km/h is kind of slow.
